src/btree_private.h

```diff
@@ -80,7 +80,7 @@ _Static_assert(offsetof(index_entry, key) == sizeof(index_entry),
  * *************************************************************************
  */
 #define MESSAGE_TYPE_BITS (2)
-_Static_assert(MESSAGE_TYPE_INVALID < (1ULL << MESSAGE_TYPE_BITS),
+_Static_assert(MESSAGE_TYPE_VALID < (1ULL << MESSAGE_TYPE_BITS),
```

Comment:
- MESSAGE_TYPE_INVALID is used to represent 0 now.
- Instead, use MESSAGE_TYPE_VALID, which represents the number of valid MESSAGE_TYPEs, to check MESSAGE_TYPE_BITS.